apps/frontend/src/components/global/upload_tabs/splunk/AuthStep.vue

```diff
@@ -59,7 +59,7 @@ import FileList from '@/components/global/upload_tabs/aws/FileList.vue';
 import {SnackbarModule} from '@/store/snackbar';
 import {LocalStorageVal} from '@/utilities/helper_util';
 import {checkSplunkCredentials} from '@mitre/hdf-converters/src/splunk-mapper';
-import {SplunkConfig} from '@mitre/splunk-sdk-no-env';
+import {SplunkConfig} from '@mitre/hdf-converters/src/converters-from-hdf/splunk/reverse-splunk-mapper';
```

Comment:
feels weird to import types and functions from the mapper for use here.  it would probably be better to make a utility file that includes these things and import from there instead.

```
hdfconverters -> src -> splunk -> utilities
                               -> mapper (depends on ./utilities)
                        converters-from-hdf -> splunk -> reverse-mapper (depends on ../../splunk/utilities)
```

Reply:
Good call. I'll centralize some commonly used functions/types among the Splunk mappers into a single utility & types file.

Tangent:

Looking at the directory tree for hdf-converters, it seems that there a bit of incoherence going on for what goes where specifically when it comes to type files. Instead of expanding it with more paths, I propose reusing and consolidating what we already have to make this more maintainable and heuristically manageable.

So the directory structure would look something like this:

hdfconverters
├── src
│   ├── converters-from-hdf
│    │   └── Splunk
│    │       └── {ONLY REVERSE MAPPER RELEVANT STUFF, MOVE TYPE FILES TO TYPES DIRECTORY}
│   └── utils
│       └── {UTILITIES GO HERE, INCLUDING UTILITIES FOR SPLUNK}
└── types
    └── {ALL TYPE DEFINING FILES GO HERE}

I'm not sure if there's a reason why a lot of these type files are in the same directory as the mappers aside from convenience, so correct me if I'm missing something.

Reply:
Works fine by me.

> I'm not sure if there's a reason why a lot of these type files are in the same directory as the mappers aside from convenience

Cause people are doing things willynilly lol